_aggregations/metric/geocentroid.md

```diff
@@ -7,99 +7,95 @@ nav_order: 45
 
 # Geocentroid
 
-The OpenSearch `geo_centroid` aggregation is a powerful tool that allows you to calculate the weighted geographic center or focal point of a set of spatial data points. This metric aggregation operates on `geo_point` fields and returns the centroid location as a latitude-longitude pair.
+The `geo_centroid` metric is a multi-value metric that calculates the weighted geographic center or focal point of a set of `geo_point` values. It returns the centroid location as a latitude-longitude pair.
```

Comment:
Is the calculation actually weighted, and if so based on what? IMO we should either reword this or specify what the weight value is.

Reply:
From the code, I don't see any weight being used, it seems just normal flat centroid calculation. So it's right to remove this word.

Reply:
```suggestion
The `geo_centroid` metric is a multi-value aggregation that calculates the geographic center or focal point of a set of `geo_point` values. It returns the centroid location as a latitude-longitude pair.
```